/**:
  ros__parameters:
    map_layer_name: "elevation"
    param_file_path: "path_default"
    map_frame: "map"
    use_sequential_load: true 
    sequential_map_load_num: 1
    use_inpaint: true
    inpaint_radius: 0.3
    use_elevation_map_cloud_publisher: false
    elevation_map_directory: "path_default"
    use_lane_filter: false
    lane_margin: 0.0
